falcon/response.py

```diff
@@ -678,6 +684,18 @@ def add_link(self, target, rel, title=None, title_star=None,
 
         """)
 
+    def set_media_type(self, media_type):
```

Comment:
It seems like this is a sort of alias for the `content_type` property. Instead of defining an additional setter method, what do you think about doing this instead:

```py
media_type = content_type
```

And then adding `media_type` under the "Attributes" section in the `Response`class docstring to note that it is simply an alias of `content_type`?

You can check how the docs looks per https://github.com/falconry/falcon/blob/master/CONTRIBUTING.md#documentation .